lispy.el

```diff
@@ -8563,14 +8563,19 @@ When ARG is non-nil, unquote the current string."
 
 (defvar lispy-mode-map-evilcp
   (let ((map (copy-keymap lispy-mode-map-base)))
-    (define-key map (kbd "M-)") 'lispy-close-round-and-newline)
+    ;; pairs
     (define-key map (kbd "[") 'lispy-open-square)
     (define-key map (kbd "]") 'lispy-close-square)
     (define-key map (kbd "{") 'lispy-open-curly)
     (define-key map (kbd "}") 'lispy-close-curly)
     (define-key map (kbd ")") 'lispy-right-nostring)
     (define-key map (kbd "\"") 'lispy-doublequote)
+    ;; wrapping
+    (define-key map (kbd "M-(") 'lispy-wrap-round)
+    (define-key map (kbd "M-{") 'lispy-wrap-braces)
+    (define-key map (kbd "M-[") 'lispy-wrap-brackets)
     (define-key map (kbd "M-\"") 'lispy-meta-doublequote)
```

Comment:
`lispy-meta-doublequote` and `paredit-meta-doublequote` are different, the second one wraps the string with double quotes, what I think is better.